easybuild/easyconfigs/h/HOME/HOME-0.9-foss-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonPackage'
+
+name = 'HOME'
+version = '0.9'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/ListerLab/HOME'
+description = """HOME (histogram of methylation) is a python package for differential methylation region (DMR) identification.
+The method uses histogram of methylation features and the linear Support Vector Machine (SVM) to identify DMRs 
+from whole genome bisulfite sequencing (WGBS) data."""
+
+toolchain = {'name': 'foss', 'version': '2017a'}
+
+source_urls = ['https://github.com/ListerLab/HOME/archive/']
+sources = ['%(version)s.tar.gz']
+patches = ['HOME-%(version)s_setup.patch']
+checksums = [
+    '6cb75a2f1fa22cd1f979fcbbaf4b7c2b1e932a6aad0b332a64013be335906266',  # 0.9.tar.gz
```

Comment:
Hmm yes me too, but for some reason --inject-checksums always returns me this checksum: 

```
== temporary log file in case of crash /tmp/eb-EOFgJ6/easybuild-crdcEt.log
== injecting sha256 checksums in /net/gmi.oeaw.ac.at/software/mendel/easybuild-home/gmi_easyconfigs/HOME-0.9-foss-2017a-Python-2.7.13.eb
== fetching sources & patches for HOME-0.9-foss-2017a-Python-2.7.13.eb...

WARNING: Found existing checksums in HOME-0.9-foss-2017a-Python-2.7.13.eb, overwriting them (due to use of --force)...

== backup of easyconfig file saved to /net/gmi.oeaw.ac.at/software/mendel/easybuild-home/gmi_easyconfigs/HOME-0.9-foss-2017a-Python-2.7.13.eb.bak_20180430090822...
== injecting sha256 checksums for sources & patches in HOME-0.9-foss-2017a-Python-2.7.13.eb...
== * 0.9.tar.gz: 6cb75a2f1fa22cd1f979fcbbaf4b7c2b1e932a6aad0b332a64013be335906266
== * HOME-0.9_setup.patch: e9f05b9cd6e3732fb6d478875f8bc821ddc6e21614eb9693a43bf7754ffbb05c
== Temporary log file(s) /tmp/eb-EOFgJ6/easybuild-crdcEt.log* have been removed.
== Temporary directory /tmp/eb-EOFgJ6 has been removed.
```

Reply:
@timeu Did you trash the `0.9.tar.gz` that EasyBuild has downlloaded before?